### PR TITLE
Feature/1053 documentation update

### LINKED ISSUE #1053: Easier and more contextualized documentation
Not being a frontend developer, I sometimes find the documentation is geared towards a skillset I haven't got.

I find mermaid to be an amazing tool but often find I need a bit more context and getting-started hints.

### PR DESCRIPTION
**Pull request by erelling**
Resolves #1053

### CODE REVIEW

**Comment by chris579** [comment]
Thanks for the pr. Make sure your pr targets `develop`, you might need to rebase your current branch. Gonna let @Dunning-Kruger look over this, he's a native speaker, unlike me.

File: docs/n00b-gettingStarted.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+2. the mermaid code we want to diagram.
+3. the `mermaid.initialize()` command to start the rendering process
+
+All this is done in the html `<body>` section of the web page.
+
+This is what needs to go into the html file:
+
+
+
+1. The reference to the mermaid renderer is done in a `<script src>` tag like so:
```
Comment (chris579) [inline docs/n00b-gettingStarted.md]: `<script>` tag

File: docs/_sidebar.md [hunk 2]
```diff
@@ -14,8 +14,15 @@
...
+
 - Guide
 
   - [Development](development.md)
   - [mermaidAPI](mermaidAPI.md)
-  - [Changelog](CHANGELOG.md)
\ No newline at end of file
+  - [Changelog](CHANGELOG.md)
+
+- I'm a n00b
```
Comment (chris579) [inline docs/_sidebar.md]: I wouldn't make a hard cut between beginners and intermediates here. Maybe the parts for beginners can be integrated into the regular docs structure.

File: docs/n00b-advanced.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# Advanced n00b mermaid (Coming soon..)
```
Comment (Dunning-Kruger) [inline docs/n00b-advanced.md]: Maybe just show it as strikethrough in the sidebar link or some other way? Clicking into a page to find out it actually is not ready is a bit annoying. If it's not done, it's better if we don't have a partial/confusing page.

Comment (erelling) [reply docs/n00b-advanced.md]: Thanks! I agree with you completely, both assessment and thoughts on rewriting. Writing from a first person perspective is not optimal in any sense, but seemed like the quickest way to jot down lessons learned. Integrating with the full docs and in a depersonalized way would be a desirable end state.

**Pull request merged**